source/beamline/BioXAS/BioXASSideTableCoordinator.cpp

```diff
@@ -388,8 +371,11 @@ void BioXASSideTableCoordinator::manipulateVerticalPVChange(double upstreamHeigh
 
 void BioXASSideTableCoordinator::manipulateHorizontalPVChange(double upstreamOffset, double downstreamOffset, AMControl* lateralPV, AMControl* yawPV)
 {
-	double lateral = ( downstreamOffset + upstreamOffset ) / 2;
-	double yaw = radianToDegree(atan((downstreamOffset - upstreamOffset) / (tableHorizontalMotorPosition_ * 2)));
+	double lateral = calculateTableLateral(upstreamOffset, downstreamOffset);
+	double yaw = calculateTableYaw(upstreamOffset, downstreamOffset);
+
+	lateral = qRound(lateral*1000)/1000.0;
```

Comment:
Hey @XuetaoLiu-CLS, as a final small question, why is it that we have to round at all?  Why can't the full floating point be passed to the PV?  Does the PV truncate it naturally and you want to make sure the precision is consistent?